app/views/pages/chooseActivity.scala.html

```diff
@@ -35,10 +35,19 @@
 
 @(journeyId: String, sicSearchForm: Form[SicSearch], chooseActForm: Form[List[SicCode]], oSearchResults: Option[SearchResults])(implicit lang: play.api.i18n.Lang, request: Request[_], messages: Messages, appConfig: AppConfig)
 
-@layout(pageTitle = Some(title(sicSearchForm, messages("page.icl.chooseActivity.title"))), fullWidth = true, backLink = true) {
+@titlePrefix = {
+    @oSearchResults.map { searchResults =>
+      @messages("page.icl.chooseActivity.resultsfound", searchResults.numFound)
+      @searchResults.currentSector.map { sector =>
+          @messages("page.icl.chooseActivity.insector", sector.name)
+      }
+    }
+}
+
+@layout(pageTitle = Some(title(sicSearchForm, titlePrefix + messages("page.icl.chooseActivity.title"))), fullWidth = true, backLink = true) {
```

Comment:
Could you separate the prefix from the rest of the title with a - (like it adds on Register for VAT and GOV.UK